examples/src/main/scala/org/apache/spark/examples/ml/LogisticRegressionSummaryExample.scala

```diff
@@ -30,11 +30,11 @@ object LogisticRegressionSummaryExample {
   def main(args: Array[String]): Unit = {
     val conf = new SparkConf().setAppName("LogisticRegressionSummaryExample")
     val sc = new SparkContext(conf)
-    val sqlCtx = new SQLContext(sc)
-    import sqlCtx.implicits._
+    val sqlContext = new SQLContext(sc)
+    import sqlContext.implicits._
```

Comment:
I don't think this import is needed anymore.  It used to be there to use the toDF() method back when there was no libsvm data source.